timesketch/frontend/src/components/Sketch/TimelineListItem.vue

```diff
@@ -168,6 +194,7 @@ import ApiClient from '../../utils/RestApiClient'
 import TsAnalyzerListDropdown from './AnalyzerListDropdown'
 import TsAnalyzerSessionDetail from './AnalyzerSessionDetail'
 import TsAnalyzerHistory from './AnalyzerHistory'
+import router from '../../router'
```

Comment:
You don't have to import the router. It is already present in the Vue instance. See my comment below for an example.